components/script/dom/htmlareaelement.rs

```diff
@@ -35,8 +189,28 @@ impl HTMLAreaElement {
                prefix: Option<DOMString>,
                document: &Document) -> Root<HTMLAreaElement> {
         Node::reflect_node(box HTMLAreaElement::new_inherited(local_name, prefix, document),
-                           document,
-                           HTMLAreaElementBinding::Wrap)
+               document,
+               HTMLAreaElementBinding::Wrap)
+    }
+
+    #[warn(dead_code)]
+    fn get_shape_from_coords(&self) -> Option<Area> {
+       let elem = self.upcast::<Element>();
+       let shp: Shape;
+       let shape = elem.get_string_attribute(&LocalName::from("shape"));
```

Comment:
get_string_attribute () method expects html5ever_atoms::LocalNameStaticSet type. But atom! ("shape") is of type servo_atoms::AtomStaticSet. 
 The same holds true for other places where you suggested us to use atom!(""). Could you please clarify this doubt?